app/controllers/report_controller/reports/editor.rb

```diff
@@ -628,6 +628,10 @@ def gfv_performance
     end
   end
 
+  def chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
@zeari what about this ?

`model.constantize < Chargeback`

then we don't have keep list of subclasses of `Chargeback` classes here.